docs/guide/search/elastic.md

```diff
@@ -0,0 +1,573 @@
+
+    API key authentication support requires that you provide the following set of parameters to
+    authenticate access to the cluster:
+
+    ``` yaml
+    <connection_name>:
+      # ...
+      authentication:
+        type: api_key
+        credentials: ['<api_key>', '<api_secret>']
+    ```
+
+    For example:
+
+    ``` yaml
+    ezplatform_elastic_search_engine:
+      connections:
+        cloud:
+          debug: true
+          elastic_cloud_id: 	'test:ZWFzdHVzMi5henVyZS5lbGFzdGljLWNsb3VkLmNvbTo5MjQzJGUwZ'
+          authentication:
+            type: api_key
+            credentials: ['8Ek5f3IBGQlWj6v4M7zG', 'rmI6IechSnSJymWJ4LZqUw']
+    ```
+
+    #### Configure SSL
+
+    When you need to protect your communication with the Elasticsearch server, you can use SSL encryption.
+    When configuring SSL for your internal infrastructure, you can use your own certificates.
+    You configure ssl by passing the path-passwords pairs for both the certificate and the certificate key.
+
+    For example:
+
+    ``` yaml
+    ezplatform_elastic_search_engine:
+      connections:
+        cloud_with_ssl:
+          debug: true
+          elastic_cloud_id: 	'test:ZWFzdHVzMi5henVyZS5lbGFzdGljLWNsb3VkLmNvbTo5MjQzJGUwZ'
+          authentication:
+            type: api_key
+            credentials: ['8Ek5f3IBGQlWj6v4M7zG', 'rmI6IechSnSJymWJ4LZqUw']
+          ssl:
+            cert:
+              path: '/path/to/cert.pem'
+              pass: ~
+            cert_key:
+              path: '/path/to/cert-key'
+              pass: ~
+    ```
+
+    After you have configured SLL, you can still disable it, for example, when the certificates expire, or you are migrating to a new set of certificates.
+    To do this, you must pass the following setting under the `ssl` key:
+
+    ``` yaml
+    verification: false
+    ```
+
+    For more information, see [Elasticsearch: SSL Encyption](https://www.elastic.co/guide/en/elasticsearch/client/php-api/7.x/security.html#_ssl_encryption_2).
+
+    #### Enable debugging
+
+    In a staging environment, you can decide that eZ Platform populates its logs with messages about the status of communication with Elasticsearch. You can then use Symfony Profiler to review the logs.
+
+    By default, debugging is disabled. To enable debugging, you can toggle either of the following two settings:
+
+    ``` yaml
+    <connection_name>:
+      # ...
+      debug:	<true/false>
+      trace:	<true/false>
+    ```
+
+    Enabling the `debug` setting results in logging basic information about a request, such as request status and time.
+    Enabling the `trace` setting results in logging additional information, such as steps to reproduce an an exact copy of a query.
+
+    !!! tip
+
+    	Make sure that you disable debugging in a production environment.
+
+    #### Configure field type mappings
```

Comment:
If this refers to Field Types as a Platform domain term, they should be capitalized (throughout the whole PR)